### PR TITLE
[Idea] proposal for standard service feature table

### PR DESCRIPTION
**Pull request by cartersocha**
I put together a rough outline for what a standard feature table could be for each service's readme. 

Comments, suggestion, & feedback please. I used this [table generator](https://www.tablesgenerator.com/markdown_tables#) to simplify the process.

### CODE REVIEW

**Review by wph95 — APPROVED** [review]

**Comment by wph95** [comment]
Just a small suggestion. Too many YES's and NO's are not very distinguishable.
Using emoji might be a good option.
YES ✅ 
NO ❌
in process 🚧

File: src/frontend/README.md [hunk 1]
```diff
@@ -9,11 +9,17 @@ else.
...
 ## OpenTelemetry features
 
-| Signal  | Instrumentation library | Custom spans | Custom span attributes |
-|---------|-------------------------|--------------|------------------------|
-| Traces  | X                       | O            | X                      |
-| Metrics | O                       | na           | na                     |
-| Logs    | O                       | na           | na                     |
+| Traces                                    | Implemented | Metrics                                   | Implemented | Logs | Implemented |
+|-------------------------------------------|-------------|-------------------------------------------|-------------|------|-------------|
+| Automatic Instrumentation (Agent)         | n/a         | Automatic Instrumentation (Agent)         | n/a         |      |             |
```
Comment (reyang) [inline src/frontend/README.md:14]: Not blocking:
1. Consider adding a link to each item with a concrete definition - e.g. what exactly does "Automatic Instrumentation (Agent)" mean.
2. Given the description + link might be long, consider changing the layout:

### Traces

* [ ] ⛔ [Automatic Instrumentation (Agent)](link)
* [ ] 🚧 ...

### Metrics

* [ ] ...

Comment (cartersocha) [reply src/frontend/README.md:14]: Nice. I like this approach

Comment (cartersocha) [reply src/frontend/README.md]: I updated the content using this structure. Not sure if the links are all perfect but it was a bit difficult to find the exact info documented.

**Review by reyang — APPROVED** [review]
LGTM. The color coding might be a problem as ~7% developers are colorblind, consider using different shapes. 🟢🟨🛑

**Review by reyang — APPROVED** [review]

**Pull request merged**